app/scripts/controllers/metamask-notifications/processors/process-feature-announcement.ts

```diff
@@ -0,0 +1,21 @@
+import type { FeatureAnnouncementRawNotification } from '../types/feature-announcement/feature-announcement';
+import type { Notification } from '../types/notification/notification';
+
+export function isFeatureAnnouncementRead(
+  notificationId: string,
+  readPlatformNotificationsList: string[],
+): boolean {
+  return readPlatformNotificationsList.includes(notificationId);
```

Comment:
Lets add the 1-month stale period for feature announcements (there is no point in showing a notification that is >1 month old). We can tweak if need to.